recipes/python-quickjs/setup.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/setup.py b/setup.py
+index 4aedcbfdda..2a4107d033 100644
+--- a/setup.py
++++ b/setup.py
+@@ -17,9 +17,6 @@ if sys.platform == "win32":
+     # 3. The code below will moneky-patch distutils to work.
+     import distutils.cygwinccompiler
+     distutils.cygwinccompiler.get_msvcr = lambda: [] 
+-    # Make sure that pthreads is linked statically, otherwise we run into problems
+-    # on computers where it is not installed.
+-    extra_link_args = ["-static"]
```

Comment:
I'm unsure, do we still need this patch?

Reply:
I would rather drop the `-static` because gcc is always called with `-shared` (see for example line 1202 of [that log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=579058&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&l=1202)). So if we remove the patch, we will execute the compiler passing simultaneously `-static` and `-shared`.